tasks/config/babel.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
       '!**/public/**',
       '!**/node_modules/**',
       '!**/bower_components/**',
-      '!**/__tests__/**'
+      '!**/__tests__/**',
+      '!**/src/forked/**'
```

Comment:
You only want `**` at the end of the pattern.  Starting with `**` means any `src/forked` path anywhere in the project will be ignored, regardless of whether it is at the root or not.

I think instead you want `'!build/kibana/src/forked/**'`